src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -41,8 +41,7 @@ function start() {
     video: false
   };
   navigator.mediaDevices.getUserMedia(constraints)
-  .then(gotStream)
-  .catch(gotStreamFailed);
+    .then(handleSuccess).catch(handleFailure);
```

Comment:
actually these should be indented by 4 right? The linter should complain here. Also decide if you want the `.` on the new line or at the end of main method line. Personally I like the `.` on the new line.